src/main/java/com/powsybl/openloadflow/sa/DcSecurityAnalysis.java

```diff
@@ -55,6 +55,7 @@ protected DcLoadFlowParameters createParameters(LoadFlowParameters lfParameters,
         var dcParameters = OpenLoadFlowParameters.createDcParameters(network, lfParameters,
                 lfParametersExt, matrixFactory, connectivityFactory, false);
         dcParameters.getNetworkParameters()
+                .setHvdcAcEmulation(false) // we only support it for DC load flow (initialization)
```

Comment:
I guess this should work in SA too ? why restricting here ?

Reply:
Done, still not supported for DC sensitivity analysis.